docs/transforms.md

```diff
@@ -0,0 +1,84 @@
+## [\[source\]](../fastai/transforms.js)
+
+## Classes
+
+* <a href="normalize">Normalize</a>
```

Comment:
Why not use markdown syntax links here? Also I don't think your anchor syntax is correct - have you tested it?

Reply:
Ooops, that one is a typo. The whole thing is only 1 page long, so didn't notice that it didn't switch page. 

May I just use Sphinx? it's very very good, and easy to use. One can simply do make html and you have searchable docs etc, and links to source, etc etc. 

Rewriting everything is really really time consuming and tiring.

Reply:
Fixed. 
Added sphinx_docs folder, to render this obsolete, so to say. 
Will remove in the future.